src/analysis/convex/topology.lean

```diff
@@ -79,21 +79,40 @@ section has_continuous_smul
 variables [add_comm_group E] [module ℝ E] [topological_space E]
   [topological_add_group E] [has_continuous_smul ℝ E]
 
+lemma convex.combo_interior_subset_interior {s : set E} (hs : convex ℝ s) {a b : ℝ}
+  (ha : 0 < a) (hb : 0 ≤ b) (hab : a + b = 1) :
+  a • interior s + b • s ⊆ interior s :=
+interior_smul₀ ha.ne' s ▸
+  calc interior (a • s) + b • s ⊆ interior (a • s + b • s) : subset_interior_add_left
+  ... ⊆ interior s : interior_mono $ hs.set_combo_subset ha.le hb hab
+
+lemma convex.combo_mem_interior {s : set E} (hs : convex ℝ s) {x y : E} (hx : x ∈ interior s)
```

Comment:
Can you provide the other side variants and make the direction clear from the name?